torch/_export/graph_module.py

```diff
@@ -291,15 +295,18 @@ def _get_submodule(
 
 
 def get_control_flow_submodules(
```

Comment:
same, what this API is for?

Reply:
For easy access of the control flow submodules? :3 Right now it is used for delegation to recurse on the submodules.

Reply:
if it's used for delegation I feel it shouldn't be included under _export/ ?